03-extract_events.py

```diff
@@ -36,10 +36,10 @@ def run_events(subject):
                                  min_duration=config.min_event_duration,
                                  shortest_event=config.shortest_event)
 
-        if config.trigger_offset:
+        if config.trigger_time_shift:
```

Comment:
@SophieHerbst this is a rename to be more explicit.

Reply:
+1